scripts/utils/climate_assessment/submit_climate_assessment.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#SBATCH --qos=priority
+#SBATCH --time=06:00:00
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=12
+#SBATCH --mem=60000
+#SBATCH --job-name=test_remind_2
+#SBATCH --output=PYTHONLOG-%x.%j.out
+source ~/.bashrc
```

Comment:
If there is anything relevant in your `~/.bashrc`, move it to this script.  Others will not have it. Others might, however, have stuff in their `~/.bashrc` that should not end up on compute nodes.